src/App.js

```diff
@@ -1,5 +1,34 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "./constants/message.js";
+import { isPlayerInputValid, isTrialInputValid } from "./utils/validation.js";
+import move from "./utils/move.js";
+import printResult from "./utils/printResult.js";
+
 class App {
-  async play() {}
+  async play() {
+    // 게임 시작
+    const playerInput = await MissionUtils.Console.readLineAsync(MESSAGES.START_INPUT);
+    const players = isPlayerInputValid(playerInput);
```

Comment:
```suggestion
    const playerInput = await MissionUtils.Console.readLineAsync(MESSAGES.START_INPUT);
    const players = playerInput.split(",").map((v) => v.trim());
    playerInputValid(players);
```
`isPlayerInputValid` 함수 이름만 봤을 때는 인풋 값을 유효성 검사해서 `boolean` 값을 리턴할 것 같은데 막상 리턴하는 값은 유저 배열이기 때문에 조금 어색한 느낌이 있는 것 같아요!

`isPlayerInputValid` 함수안에 있는 `const players = input.split(",").map((v) => v.trim());` 이 로직을 밖을 빼고 `players`를 `isPlayerInputValid` 인자에 넣는건 어떨까요?? 

그리고 내부에서 `boolean` 값을 리턴하는 방식을 사용하거나 `isPlayerInputValid`의 함수 이름을 `playerInputValid`로 수정해도 좋을 것 같아요!

Reply:
오! 이게 훨씬 함수의 기능을 정확하게 할 수 있겠네요! 감사합니다 👍